<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KisDlgCustomTabletResolution</class>
 <widget class="QDialog" name="KisDlgCustomTabletResolution">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Advanced Tablet Setting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Select area your tablet is mapped to:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioMapAsWintab">
        <property name="text">
         <string>Use information provided by tablet</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioMapToEntireScreen">
        <property name="text">
         <string notr="true">Map to entire virtual screen NO_I18N</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="radioCustomMapping">
          <property name="text">
           <string>Map to custom area</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" rowspan="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="lblWidth">
            <property name="text">
             <string>Width:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QSpinBox" name="intWidth">
            <property name="suffix">
             <string>px</string>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="lblXOffset">
            <property name="text">
             <string>X offset:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QSpinBox" name="intXOffset">
            <property name="suffix">
             <string>px</string>
            </property>
            <property name="minimum">
             <number>-10000</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="lblHeight">
            <property name="text">
             <string>Height:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="intHeight">
            <property name="suffix">
             <string>px</string>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="lblYOffset">
            <property name="text">
             <string>Y offset:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QSpinBox" name="intYOffset">
            <property name="suffix">
             <string>px</string>
            </property>
            <property name="minimum">
             <number>-10000</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lblDisplayInformation">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="btnBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
